platform/security/sts/security-sts-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java

```diff
@@ -89,12 +89,13 @@ public AttributeQueryClaimsHandler() {
    * @return List of supported claims.
    */
   @Override
-  public List<URI> getSupportedClaimTypes() {
+  public List<String> getSupportedClaimTypes() {
```

Comment:
This one was straight from the 3.3.3 migration guide:
```The claimType of the Claim class is now a "String" instead of a "URI".  This might break existing ClaimsHandler implementations in the STS. In addition, the ClaimsHandler interface now returns a List<String> for getSupportedClaimTypes() instead of List<URI>.```

In all the uses of claimType, I kept the conversion to URI, then did toString(), to keep the URI validation and exception handling in place (don't want a bad URI to make it through because it is simply a string field now).

Reply:
These are all going to get turned to strings once I put up my PR. Just fyi.